.eslintignore

```diff
@@ -0,0 +1,3 @@
+bundle
```

Comment:
I added `test/app/dist` but I don't understand what are the "sandbox files that are not versioned", can you explain please?

Reply:
sandbox directory is used for local test and come with a default index.html.
however, you can add your own test resources in it  which would be served by the dev server.

Extra files added to this directory are not versioned, cf my workspace:
<img width="212" alt="Capture d’écran 2021-01-14 à 16 16 42" src="https://user-images.githubusercontent.com/1331991/104609968-e6ee1800-5683-11eb-8b5f-342daaf1c01d.png">

It could be nice to not lint them to avoid unnecessary errors locally

Reply:
Thank you for the clarification.

In that case we can ignore the `sandbox` directory altogether since it doesn't contain any JS/TS file, right?